### PR TITLE
Convert tutorials from .rst to .md

### PR DESCRIPTION
**Pull request by zupo**
With this done, we no longer need the `rst2myst` command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<img alt="Netlify Status"
+     src="https://api.netlify.com/api/v1/badges/269f7467-6afd-49ae-97f2-61a160e93a9a/deploy-status">
```
Comment (zupo) [inline README.md:2]: This url is broken (404), @domenkozar can you get a working one pls?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+<img alt="CI"
+     src="https://github.com/nix-dot-dev/nix.dev/workflows/CI/badge.svg">
```
Comment (zupo) [inline README.md:4]: @domenkozar: do you want this image to link to somewhere?

File: README.md (hunk 1 above)
Comment (domenkozar) [reply README.md:2]: let's just remove it, using cloudflare pages now that don't have a badge

File: README.md (hunk 2 above)
Comment (domenkozar) [reply README.md:4]: nah :)

**Comment by domenkozar** [comment]
@zupo is this ready? (it's in draft mode)

File: README.md (hunk 1 above)
Comment (zupo) [reply README.md]: Removed.

**Pull request merged**